apps/web/src/pages/forms/ActivitiesGap.tsx

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
The path says `pages/forms` -> could you move this to the components folder, because anything in the pages section will have a route. Also instead of forms -> Planning. Forms in the frontend would mean the <form> or something similar to that.